packages/teleport/src/Apps/AddApp/AddApp.tsx

```diff
@@ -22,7 +22,7 @@ import * as Icons from 'design/Icon';
 
 import useTeleport from 'teleport/useTeleport';
 
-import { TabIcon } from 'teleport/components/Tabs';
+import TabIcon from 'teleport/components/Tabs/TabIcon';
```

Comment:
Importing the named export doesn't appear to be an issue on my machine.

But [I've run into a similar problem](https://gravitational.slack.com/archives/C02DQ1C2BMW/p1667223568113269) with webpack today. Could you check if removing the `packages/teleport/node_modules/.cache` directory helps?

Reply:
Yeah that fixed it.